web/Navigation.tsx

```diff
@@ -11,6 +11,7 @@ import {
   MailPlus,
   MessageCircle,
   Settings,
+  ShoppingBag,
```

Comment:
why is it called a ShoppingBag?

Reply:
that's the icon for the chub tab, from lucide-solid
![image](https://github.com/agnaistic/agnai/assets/26354814/369a2566-cf2c-49cf-ac19-135a9a386238)